Source/WTF/wtf/cocoa/MemoryPressureHandlerCocoa.mm

```diff
@@ -61,7 +61,7 @@
     return source.get();
 }
 
-static int notifyTokens[3];
+static int notifyTokens[6];
```

Comment:
Could we provide some more details around where the number 6 comes from and what it represents.